extensions/amp-image-slider/1.0/base-element.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {PreactBaseElement} from '#preact/base-element';
+
+import {ImageSlider} from './component';
+import {CSS as COMPONENT_CSS} from './component.jss';
+
+export class BaseElement extends PreactBaseElement {}
+
+/** @override */
+BaseElement['Component'] = ImageSlider;
+
+/** @override */
+BaseElement['props'] = {
+  images: {selector: 'img' || 'amp-img', single: false, clone: true},
+  labels: {selector: 'div', single: false, clone: true},
+  initialPosition: {
+    attr: 'initial-slider-position',
+    type: 'string',
+  },
+  shouldHintReappear: {
+    attr: 'disable-hint-reappear',
```

Comment:
Hi, I have a question regarding this.

Changing this would not be backward compatible with AMP version.
Here, `disable-hint-reappear` attribute is specified in the [Markdown](https://github.com/ampproject/amphtml/blob/main/extensions/amp-image-slider/amp-image-slider.md#disable-hint-reappear).

Example Usage:
https://github.com/ampproject/amphtml/blob/cfb102aafc1543788b77f3ef9bd138c753e13097/examples/image-slider.amp.html#L186-L187
https://github.com/ampproject/amphtml/blob/cfb102aafc1543788b77f3ef9bd138c753e13097/examples/image-slider.amp.html#L219-L220

Note: If, we rename this attribute, we also need to update Markdown (to add this modified attribute for Bento Version).

Should I rename it?

Reply:
Renaming it and updating the markdown for the `1.0` version is fine. The intention is to not have an attribute with a negative name, since enabling or disabling it can be confusing to reason about. On second thought, `repeat-hint` is not a good alternative because it flips the default behavior when omitted. Let's rename to `display-hint-once` or similar, which has the same default omitted/provided behavior, but perhaps a clearer name. Open to suggestions here if you have any!

Reply:
Very insightful💡

`display-hint-once` seems pretty straight forward. I will update accordingly including markdown for `1.0`.